app/models/user.rb

```diff
@@ -80,4 +124,17 @@ def daac_curator?
   def arc?
     admin || curator
   end
+
+  private
+
+  def get_user_info
```

Comment:
same question about a http/api call in the model. I think methods in models should have tests. I also think that api calls (in general but not necessarily for all cases) should be be mocked, so if this method is kept in the model we can either stub it or move your `return true if Rails.env.test?` line here.

Reply:
Same response as above...